services/QuillLMS/spec/models/concept_feedback_spec.rb

```diff
@@ -74,39 +75,116 @@
       it 'adds the translations to the data' do
         expect(subject["translatedDescription"]).to eq(translation)
       end
+
+      context 'there is only a gengo translation' do
+        let(:translated_text) { create(:gengo_translated_text, translation: translation)}
```

Comment:
Could be `create(:gengo_translated_text, translation:)`

Reply:
I'd like to find a linter that would do this for us automatically!

Reply:
I will make another PR to add this to Rubocop
```Style/HashSyntax:
  EnforcedShorthandSyntax: consistent```

Reply:
Good idea.  Please add this